### PR TITLE
Setting up seeder

### PR DESCRIPTION
**Pull request by darjama**
-added imageList to generate image URLs
-added pgSeed.js to create .csv files to insert into database
-added importCsv.js to bring .csv files into database

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -33,8 +33,14 @@ app.put('/products/:productId', (req, res) => {
...
 });
 
+<<<<<<< HEAD
+app.delete('/products', (req, res) => {
+  console.log('req', req);
+  models.deleteProduct(req.query.productId, (err, results) => {
+=======
 app.delete('/products/:productId', (req, res) => {
   models.deleteProduct(req.params.productId, (err, results) => {
+>>>>>>> 033cd84b675e660067a29f21d72fed179cea34f0
```
Comment (briansjkim) [inline server/index.js]: Looks like there are some merge conflicts if you haven't gotten to them yet!

File: server/seed/pgSeed.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      const randomNum = (max) => Math.round(Math.random() * max);
+      const randomNumUpTo4 = randomNum(4);
+      for (let x = 0; x < randomNumUpTo4 + 1; x += 1) {
+        productArray.push(imagePrefix + images[randomNum(imagesLength)]);
+      }
+      const picArray = `"{${productArray.join(',')}}"`;
+      buffer += `${i}, ""${randomProductItem}"", ${picArray},${i % 3 === 0}\n`;
+      // models.saveProduct(i, randomProductItem, productArray, likeArray[randomNumUpTo2]);
+    }
+    const result = await fs.writeFileSync(`./csv/pgProductImport${1000 + j}.csv`, buffer);
```
Comment (briansjkim) [inline server/seed/pgSeed.js:24]: Nice use of async/await here. I couldn't figure out how to incorporate it with npm csv-writer, but looks great!

**Review by sungjinbkim — CHANGES_REQUESTED** [review]

File: server/seed/pgSeed.js (hunk 2 above)
Comment (darjama) [reply server/seed/pgSeed.js:24]: It's a bit of a kludge. Generally, we're not supposed to use async inside a loop, and result is undefined, but it seems to work.

File: server/index.js (hunk 1 above)
Comment (darjama) [reply server/index.js]: Thanks, fixed them if you get a chance to review again.

**Pull request merged**